src/contracts/campaigns/BraidCampaign.sol

```diff
@@ -0,0 +1,195 @@
+pragma solidity ^0.4.4;
+
+
+import "utils/Owned.sol";
+import "verifiers/OwnedVerifier.sol";
+import "tokens/StandardToken.sol";
+
+/*
+A custom WeiFund compatable campaign for the Braid the Movie Ethereum campaign.
+*/
+
+contract BraidCampaign is Owned, StandardToken {
+  modifier pastFreezeDate {
+    // now must be greater than freeze date
+    if (block.number >= freezePeriod) {
```

Comment:
Yes, the only reason I did this was to keep the interface the same. We can change, but JS will have to be changed also.